batch7crud-roll3/src/main/java/com/lftechnology/batch7crud/db/DBConnection.java

```diff
@@ -0,0 +1,37 @@
+package com.lftechnology.batch7crud.db;
+
+import java.sql.SQLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.sql.DataSource;
+import com.lftechnology.batch7crud.exception.DataException;
+import java.sql.Connection;
+
+public class DBConnection {
+	private static Logger logger = Logger.getLogger("DBConnection");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make the "logger" logger private static final and rename it to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1312)